lib/tasks/db.rake

```diff
@@ -157,8 +157,10 @@ task 'multisite:migrate' => ['db:load_config', 'environment', 'set_locale'] do |
   SeedFu.seed(seed_paths, /001_refresh/)
 
   execute_concurently(concurrency, exceptions) do |db|
-    puts "Seeding #{db}"
-    SeedFu.seed(seed_paths)
+    if ENV['SKIP_SEED'] != '1'
+      puts "Seeding #{db}"
+      SeedFu.seed(seed_paths)
```

Comment:
Actually can we fold this into the `Discourse.skip_post_deploy_migrations` check below?